students/MicahBraun/Lesson 03/mailroom.py

```diff
@@ -0,0 +1,227 @@
+# ------------------------------------------------------------------------
+# NAME: MICAH BRAUN
+# PROJECT: mailroom.py
+# PURPOSE: Working with arraysv(nested lists), manipulating data
+# DATE: 05/24/2018
+#
+# DESCRIPTION: Program is menu-driven user interface for interacting
+#              with a nested list of stored information. User is able
+#              to add names to a list (not currently saved upon program
+#              exit), append values to fields, check against current
+#              fields, create mock "Thank You" e-mails based on user-
+#              -input and associated fields, and generate tabular display
+#              of data for viewing.
+# ------------------------------------------------------------------------
+
+# Data -------------------------------------------------------------------
+spacing = '- ' * 55  # formatting for DONOR header
+data_list = [['NAMES', 'DONATION AMOUNT', 'NUMBER OF GIFTS', 'AVG. GIFTS'],
+             ['Rudolph Soares', 1500, 3, 1566],                                    # Donor info
+             ['Josef Mistretta', 250, 5, 235],
+             ['Joye Agostini', 5000, 2, 5750],
+             ['Joni Mattix', 2750, 1, 2750],
+             ['Rachelle Levan', 750, 3, 581],
+             ['Vena Ussery', 1000, 7, 785],
+             ['Efrain Lager', 10000, 1, 10000],
+             ['Mee Heine', 15000, 2, 9750],
+             ['Tanya Essex', 50000, 1, 50000],
+             ['Garrett Hartsell', 800, 2, 1400]]
+
+search_names = [elem[0] for elem in data_list]
+
+# Processing --------------------------------------------------------------
+
+def menu():
+    """Display menu of options to user"""
+    header = "-" * 100  # formatting for header
+    title = ' ' * 43 + "MAIL ROOM MENU"  # title display
+    directions = "Select one of the three options:"  # directions to user for menu
+    optA = "[A] Send a Thank you"  # option(s) for menu
+    optB = "[B] Create a Report"
+    optC = "[C] Quit"
+    whole_menu_header = header + '\n' + title + '\n' + header  # header strung together as one uni
+
+
+    while True:
+        print(whole_menu_header)  # display header
+        print(directions.center(100, ' '), '\n')  # center directions with header display
+        print(optA + optB.center(60, ' ') + optC.center(31, ' ') + '\n')
+
+        usr_sel = str(input("Menu Selection: ")).upper()
+
+        if usr_sel == "A":
+            print("Leaving menu...")
+            thankyou()
+        elif usr_sel == "B":
+            print("Leaving menu...")
+            createreport()
+        elif usr_sel == "C":
+            input("Exiting Program...[Press Enter]")
+            exit()
+        elif usr_sel != "A" or usr_sel != "B" or usr_sel != "C":
+            print("\nOnly enter A, B, or C!")
+            continue
```

Comment:
since you already checked if user entered A or B or C, you only need an `else` here

also, no need to `continue` it will naturally continue